.gitlab-ci.yml

```diff
@@ -72,5 +77,18 @@ release:
     - make deps
     - poetry config pypi-token.pypi $PYPI_PASSWORD
     - make publish
+
+
+release-github:
+  stage: release
+  artifacts:
+    paths:
+      - dist/
+    when: always
+  only:
+    - /^v[0-9]+\.[0-9]+\.[0-9]+.*/
+  except:
+    - branches
+  script:
     - shasum -a 256 dist/* > dist/checksums.txt
```

Comment:
I think checksum should be calculated by pypi job as well and become part of the pypi publish job output.

Reply:
Moved to the previous job.